packages/app-extensions/src/actions/modules/actionHandlers/legacyAction.js

```diff
@@ -48,6 +49,11 @@ export default function* (definition, selection, parent, params, config) {
 
   const actionDefinition = new window.nice2.netui.actions.model.ClientActionDefinition(definition.id, definition.id)
   actionDefinition.setEnabled(!definition.readOnly)
+  if (_isPlainObject(definition.properties)) {
+    Object.keys(definition.properties).forEach(propertyName => {
```

Comment:
(purely a matter of taste) `Object.entries(definition.properties).forEach(([key, value]) => actionDefinition.setProperty(key, value)) ` would save you the bracket property notation.